README.md

```diff
@@ -16,16 +15,19 @@ import * as ActionTypes from '../constants/ActionTypes';
 
 const initialState = [];
 
-
 export const todos = createReducer(initialState, {
   [ActionTypes.ADD_TODO](state, action) {
-    let text = action.text.trim();
+    const text = action.text.trim();
     return [...state, text];
   }
-})
-```
 
+  [ActionTypes.REMOVE_TODO](state, action) {
+    return [...state.filter((_, i) => i !== action.index)];
```

Comment:
I believe that filter already returns a new array, so there's no need for the spread operator here

Reply:
Good shout. I don't think we need the square bracket parenthesis either then, otherwise we'll get an array of an array. I'll modify and push now.

Reply:
Test code:
```js
const arr = [1];
const arr2 = [...arr, 2];
const arr3 = arr2.filter((_, i) => i !== 1);
console.log(arr, arr2, arr3);
```